decidim-core/app/packs/src/decidim/input_mentions.js

```diff
@@ -5,8 +5,7 @@ const mentionsInitializer = () => {
   const $mentionContainer = $(".js-mentions");
   const nodatafound = $mentionContainer.attr("data-noresults");
 
-  // The editor implements hashtags functionality by itself so it is not needed
-  // to attach tribute to the rich text editor.
+  // Used to render mentions in the rich text editor.
```

Comment:
This comment is misleading, as it isn't refererring why the guard clause is needed

Reply:
Updated description.